components/forms/README.md

```diff
@@ -0,0 +1,19 @@
+## Forms
+
+Forms, i.e. written denotations of the linguistic sign (cf. [GOLD's FormUnit](http://linguistics-ontology.org/gold/2010/FormUnit)), are stored in a
+`FormTable` in CLDF datasets (typically `Wordlist`s).
+
+ Each form is stored as a separate row in this table, minimally providing values
```

Comment:
Each form is stored as a separate row in this table, providing, at the least, the following columns: